magda-preview-map/wwwroot/init/terria.json

```diff
@@ -5,5 +5,6 @@
         "south": -45,
         "west": 109
     },
+    "viewerMode": "2d",
```

Comment:
Maybe let's leave it as 3D on desktop? Especially seeing how it still goes to 3D if the width is too low like I showed you :)